.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -16,7 +16,16 @@
 
 param([array] $Arguments)
 
-foreach ($project in (dotnet sln list | Select-Object -Skip 2 | Get-Item))
+if (!(Get-ChildItem *.sln))
+{
+    $projects = Get-ChildItem *.csproj
+}
+else
+{
+    $projects = (dotnet sln list | Select-Object -Skip 2 | Get-Item)
+}
```

Comment:
In PowerShell statements return a value too. This is so beautiful it brings tears to my eyes.
So you can do
```pwsh
$projects = if ( ... )
{
}
else
{
}
```
instead of assigning to the variable in each branch.

Reply:
😂 or 😢 ?

Reply:
🥲

Reply:
Seems like poor man's ternary operator to me! Fortunately, PS 7 has actual ternary operators. I briefly contemplated keeping Windows PowerShell compatibility.

Reply:
It's more like a rich man's ternary operator because you can have multiple expressions in each arm so it's more powerful than a ternary. But yes, please use ternary then. This repo is [already not compatible](https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/actions/setup-sql-server/Initialize-SqlServer.ps1#L7-L9) with Windows PowerShell.

Reply:
Indeed, and it's lavish with all the characters. Almost like showing off! :D

Reply:
![image](https://user-images.githubusercontent.com/4007293/183134904-9b58ec55-fc98-4dd7-a763-7eab28c80831.png)

Reply:
:D